docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "2"
+services:
+  dev:
+    build: .
+    image: keboola/ex-http-dev
+    tty: true
+    stdin_open: true
+    environment:
+        - KBC_DATADIR=/code/data/
+    volumes:
+        - ./:/code
+        - ./data:/data
+
+  prod:
+    build: .
+    image: keboola/ex-http
+    environment:
+        - KBC_DATADIR=/code/data/
+    volumes:
+        - ./data:/data
```

Comment:
tohle je tady imho zbytečný. můžu zkusit nějak víc rozepsat proč.

Reply:
úplně celé jsem to pročistil a nechal jen ten compose na dev a travis udělal normálním dockerem.